package.json

```diff
@@ -732,6 +732,11 @@
         "title": "Text Editor",
         "order": 1,
         "properties": {
+          "dotnet.autoInsert.enableAutoInsert": {
```

Comment:
Will auto insert be used for more than doc comments? Perhaps the settings should be `dotnet.autoInsert.insertDocComments` imagining that certain auto inserts might be disabled without disabling all auto inserts.

Reply:
If we do just want one big switch, then I would suggest `dotnet.autoInsert.enabled`